src/jobs/generation/MetaGenerator.groovy

```diff
@@ -329,7 +329,7 @@ repos.each { repoInfo ->
 
                         if (isDSLPRFromSameRepo) {
                             if (isVSTS) {
-                                // TODO: VSTS PR refspec
+                                refspec('+refs/pull/${ghprbPullId}/merge:refs/remotes/pull/${ghprbPullId}/merge')
```

Comment:
This refspec is wrong.  The variable ghprbPullId wouldn't be valid for VSTS here (it's set by the github pull request builder)

Reply:
As discussed, changed it to vstsRefspec, which is set by TFS Jenkins plugin depending on whether this is is a PR build.